tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/GitHubWebhookProcessor.cs

```diff
@@ -57,12 +57,18 @@ await Policy
                     switch (ex)
                     {
                         case AbuseException abuseException:
-                            logger.LogWarning("Abuse exception detected, attempting retry.");
+                            logger.LogWarning("Abuse exception detected. Retry after seconds is: {retrySeconds}",
+                                abuseException.RetryAfterSeconds
+                                );
                             retryDelay = TimeSpan.FromSeconds((double)abuseException.RetryAfterSeconds);
                             break;
 
                         case RateLimitExceededException rateLimitExceededException:
-                            logger.LogWarning("Rate limit exception detected, attempting retry.");
+                            logger.LogWarning(
+                                "Rate limit exception detected. Limit is: {limit}, reset is: {reset}",
+                                rateLimitExceededException.Limit,
+                                rateLimitExceededException.Reset
+                                );
```

Comment:
Should you also log GetRetryAfterTimeSpan?

Reply:
Added